src/FileMetadata/typings/index.ts

```diff
@@ -0,0 +1,69 @@
+import { LearningObject } from '../../shared/entity';
+export type LearningObjectFile = LearningObject.Material.File;
+
+/**
+ * An interface representation of the FileMetadataDocument type
+ *
+ * @export
+ * @interface FileMetadataDocument
+ */
+export interface FileMetadataDocument {
+  /**
+   * Unique identifier of the file document
+   */
+  id: string;
+  /**
+   * Time at which the file metadata was created (When the file was first ever uploaded)
+   * Value is a string representing the milliseconds elapsed since the UNIX epoch
+   */
+  createdDate: string;
+  /**
+   * User inputted information about the contents of the file
```

Comment:
My editor yells at me for anything that is not standard English -what about this? 

```suggestion
   * User supplied information about the contents of the file
```